presto-spi/src/main/java/io/prestosql/spi/security/PrincipalType.java

```diff
@@ -15,5 +15,5 @@
 
 public enum PrincipalType
 {
-    USER, ROLE
+    USER, ROLE, GROUP
```

Comment:
```suggestion
    USER, 
    ROLE, 
    GROUP,
    /**/
```

@martint I think we were here someday.  See https://github.com/prestosql/presto/pull/2940